### PR TITLE
remove_{prefix,suffix} returns the original string if the param is not a prefix/suffix

### PR DESCRIPTION
**Pull request by matthieu-foucault**
(gsub! returns nil when no substitution is performed)

### CODE REVIEW

**Comment by bbatsov** [comment]
This should also be mentioned in the changelog.

**Comment by matthieu-foucault** [comment]
I updated the changelog.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 ## master (unreleased)
 
+* Changed `String#remove_prefix` returns the original string if the parameter is not a prefix
```
Comment (bbatsov) [inline CHANGELOG.md]: to return

**Comment by matthieu-foucault** [comment]
Fixed the changelog

**Comment by bbatsov** [comment]
Had to apply your patch manually, but it's finally in. Sorry about forgetting about it!

**Pull request closed without merging**